modules/installation-special-config-butane.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-butane_{context}"]
+= Creating machine configs with Butane
+
+Machine configs are used to deploy control plane and worker machines by telling machines how to create users and filesystems, set up the network, install systemd units, and more.
+
+Because modifying machine configs can be difficult, you can use Butane configs to create machine configs for you, thereby making node configuration much easier.
+
+== About Butane
+
+Butane is a YAML utility that {product-title} uses to provide convenient short-hand syntax for writing machine-readable machine configs, as well as for performing additional validation of  machine configs. The format of the Butane config file that Butane transpiles is defined in the
+https://coreos.github.io/butane/config-openshift-v4_8/[OpenShift Butane config spec].
+
+[id="rhcos-installing-butane_{context}"]
+== Installing Butane
+
+You can install the Butane tool (`butane`) to create {product-title} machine configs from a command-line interface. You can install `butane` on Linux, Windows, or macOS by downloading the related binary file.
+
+[TIP]
+====
+Butane releases are backwards-compatible with older releases and with the Fedora CoreOS Config Transpiler (FCCT).
+====
+
+.Procedure
+
+. Navigate to the Butane image download page at https://mirror.openshift.com/pub/openshift-v4/clients/butane/.
+. Get the `butane` binary:
+.. For the newest version of Butane, save the latest `butane` image to your current directory:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane --output butane
+----
++
+.. Optional: For a specific type of architecture you are installing Butane on, such as aarch64 or ppc641e, indicate the appropriate URL. For example:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane-aarch64
+----
+
+. Move the `butane` binary file to a directory on your `PATH`.
++
+To check your `PATH`, open a terminal and execute the following command:
++
+[source,terminal]
+----
+$ echo $PATH
+----
++
+. Make the downloaded binary file (for example, `butane`) executable:
++
+[source,terminal]
+----
+$ chmod +x butane
+----
++
+You can now use the Butane tool by running the `butane` command:
++
+[source,terminal]
+----
+$ butane <butane_file.bu>
+----
+
+[id="rhcos-creating-with-butane_{context}"]
+== Creating a `MachineConfig` object by using Butane
+
+By using Butane, you can produce a `MachineConfig` object so that you can configure worker or master nodes at installation time or via the Machine Config Operator.
+
+.Prerequisites
+
+* You have installed the `butane` utility.
+
+.Procedure
+
+. Create a Butane config YAML file. The following example tells the {product-title} cluster to use a real-time kernel for all worker nodes:
++
+[source,terminal]
```

Comment:
For improved syntax highlighting, I've been putting the filename (`99-worker-realtime.bu`) in the bullet text above, then using `[source,yaml]` here and dropping the `cat`/`EOF` lines.

Reply:
Cool, I originally had `[source,yaml]` but talked myself into combining steps. I've edited to more closely match your `99-simple-kmod.bu` example.